### PR TITLE
Fix case in InetAddress.toString() that truncates last group.

### PR DESCRIPTION
**Pull request by tolbertam**
For [NODEJS-255](https://datastax-oss.atlassian.net/browse/NODEJS-255)

### CODE REVIEW

File: lib/types/inet-address.js [hunk 1]
```diff
@@ -147,12 +147,16 @@ InetAddress.prototype.toString = function (encoding) {
       if (start < 0) {
         start = i;
       }
+
+      // at the end of the buffer, make a final call to checkLongest.
+      if(i == this.buffer.length - 2) {
```
Comment (jorgebay) [inline lib/types/inet-address.js]: Use strict equality `===` :)

**Comment by jorgebay** [comment]
Nice catch!
Minor nit about the operator, otherwise looks good to me.

File: lib/types/inet-address.js (hunk 1 above)
Comment (tolbertam) [inline lib/types/inet-address.js]: d'oh!  Keep making that mistake...will fix shortly.

Comment (tolbertam) [inline lib/types/inet-address.js]: Fixed and squashed, thanks for pointing that out :)

**Pull request merged**